easybuild/easyblocks/q/qscintilla.py

```diff
@@ -86,32 +86,43 @@ def install_step(self):
             except OSError as err:
                 raise EasyBuildError("Failed to change to %s: %s", pydir, err)
 
+            pyqt5 = get_software_root('PyQt5')
+            pyqt = get_software_root('PyQt')
+            if pyqt5:
+                self.pyqt_root = pyqt5
+                self.pyqt_version = "PyQt5"
+            elif pyqt:
+                self.pyqt_root = pyqt
+                self.pyqt_version = "PyQt4"
+            else:
+                raise EasyBuildError("Failed to determine PyQt installation prefix, PyQt/PyQt5 not included as dependency?")
+
             # apparently this directory has to be there
-            qsci_sipdir = os.path.join(self.installdir, 'share', 'sip', 'PyQt4')
+            qsci_sipdir = os.path.join(self.installdir, 'share', 'sip', self.pyqt_version)
             mkdir(qsci_sipdir, parents=True)
 
-            pylibdir = os.path.join(det_pylibdir(), 'PyQt4')
-
-            pyqt = get_software_root('PyQt')
-            if pyqt is None:
-                raise EasyBuildError("Failed to determine PyQt installation prefix, PyQt not included as dependency?")
+            pylibdir = os.path.join(det_pylibdir(), self.pyqt_version)
 
             cfgopts = [
                 '--destdir %s' % os.path.join(self.installdir, pylibdir),
                 '--qsci-sipdir %s' % qsci_sipdir,
                 '--qsci-incdir %s' % os.path.join(self.installdir, 'include'),
                 '--qsci-libdir %s' % os.path.join(self.installdir, 'lib'),
-                '--pyqt-sipdir %s' % os.path.join(pyqt, 'share', 'sip', 'PyQt4'),
+                '--pyqt-sipdir %s' % os.path.join(self.pyqt_root, 'share', 'sip', self.pyqt_version),
                 '--apidir %s' % os.path.join(self.installdir, 'qsci', 'api', 'python'),
                 '--no-stubs',
+                '--no-dist-info',
             ]
+            if pyqt5:
+                 cfgopts.append("--pyqt=PyQt5")
```

Comment:
good remark, according to the docs it would, but this flag was only added in version 2.11.*.
Previous version didn't have it AFAIK. This will need some additional logic there.